materialize-bigquery/bigquery.go

```diff
@@ -78,13 +78,11 @@ func (c *config) client(ctx context.Context) (*client, error) {
 	if err != nil {
 		return nil, fmt.Errorf("creating bigquery client: %w", err)
 	}
-	log.WithField("projectID", billingProjectID).Info("bigquery client successfully created")
```

Comment:
Cleaning up some extraneous logging in this file that was added in https://github.com/estuary/connectors/pull/551 which I always intended to remove but never got around to until now.